deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -511,4 +560,23 @@ spec:
     port: 24321
     targetPort: 24321
     protocol: TCP
+  - name: metrics
+    port: 24231
+    targetPort: 24231
+    protocol: TCP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: fluentd-events
+  labels:
+    k8s-app: fluentd-sumologic-events
+spec:
+  selector:
+    k8s-app: fluentd-sumologic-events
+  ports:
+  - name: metrics
+    port: 24231
+    targetPort: 24231
```

Comment:
Not needed for now, but we should look at using named ports for the targetPort so we dont have to update the Pod and Service each time.